manifests/05-operator.yaml

```diff
@@ -16,12 +16,13 @@ spec:
       serviceAccountName: console-operator
       containers:
       - name: console-operator
-        image: openshift/origin-console-operator:latest
+        image: docker.io/openshift/origin-console-operator:latest
         ports:
         - containerPort: 60000
           name: metrics
         command:
         - console-operator
+        args: ["-create-default-console=true"]
```

Comment:
`oc create -f manifests/` will now deploy an operator, which will automatically create a console if it does not exist.

Reply:
I'll do that, agree!